evap/results/exporters.py

```diff
@@ -101,11 +106,18 @@ def filter_text_and_heading_questions(questions):
         return filtered_questions
 
     @staticmethod
-    def filter_evaluations(semesters, evaluation_states, degrees, course_types, contributor, include_not_enough_voters):
+    def filter_evaluations(
+        semesters: Iterable[Semester],
+        evaluation_states: Iterable[int],
+        degrees: Iterable[int],
+        course_types: CourseType,
```

Comment:
Can you rename these to `degree_ids` and `course_type_ids` throughout the exporters?